ros2_batch_job/__main__.py

```diff
@@ -86,6 +86,9 @@ def get_args(sysargv=None):
         '--repo-file-url', required=True,
         help="url of the ros2.repos file to fetch and use for the basis of the batch job")
     parser.add_argument(
+        '--supplemental-repo-file-url', default=None,
+        help="url of a .repos file to fetch and merge with the ros2.repos file")
```

Comment:
Multiple files would be cool, but I think it's fine to wait and do that when we require it.